gax-grpc/src/test/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProviderTest.java

```diff
@@ -343,4 +343,14 @@ public ManagedChannelBuilder apply(ManagedChannelBuilder channelBuilder) {
 
     provider.getTransportChannel().shutdownNow();
   }
+
+  @Test
+  public void testEpollAvailable() throws Exception {
```

Comment:
This is testing that epoll is available when gax runs tests. Should this be a runtime check?

Reply:
Yes it's a runtime checker. So you mean this test is not helping? I just want to make sure when channel gets created, it will be using the epoll channel.

Reply:
Yeah, this is testing that epoll channel is available when gax runs it's tests. If it's a transitive dependency of gax then this might help but it's not guaranteed. It's possible that the dependency is being pulled in by a test dependency.

What happens if epoll is not available?

Reply:
That makes sense. I'll just remove this test. AFAICT, since gax already depends on grpc-netty-shaded, as long as the client library uses gax to create channel, they will most likely be using epoll.

If epoll is not available, say some client libraries use their own channel builder, or for a non-linux machine, then this keepAliveTimout value will not be used to set the TCP_USER_TIMEOUT socket option, because that socket option is only supported by epoll. https://github.com/grpc/grpc-java/blob/master/netty/src/main/java/io/grpc/netty/NettyClientTransport.java#L230-L233